unity-renderer/Assets/Rendering/Outliner/Shaders/GaussianBlur.cginc

```diff
@@ -2,9 +2,9 @@
 #define OUTLINER_BLUR
 #define OUTLINER_PI 3.14159265
 
-//#define KERNEL_SIZE 35
-#define KERNEL_SIZE 127
-//#define KERNEL_SIZE 7
+#define KERNEL_SIZE_LOW 4
+#define KERNEL_SIZE_MID 16
+#define KERNEL_SIZE_HIGH 32
```

Comment:
We could avoid the define and set an uniform for this: `_GaussianBlurKernelSize` to be set through the material. It would make everything cleaner.

Reply:
I also see you are using a Shader.SetGlobal for this index, we could set manually the steps there haha.

Reply:
Yeah, It was like this at first but I separated them to do some mad science but I ended up doing nothing 😆, rolled back!